src/App.js

```diff
@@ -1,11 +1,27 @@
 import React from 'react';
 import Navbar from "./components/Navbar/Navbar";
 import './App.css';
+import { BrowserRouter } from 'react-router-dom';
+import { BrowserRouter as Router, Route, Switch, Link } from "react-router-dom"
+import Home from "./pages/Home/Home";
+import Agency from "./pages/Agency/Agency";
+import AgencyProfileForm from "./pages/agency-profile-form/AgencyProfileForm";
```

Comment:
Nick's PR will change the directory structure of our project, so sometime in the near future we will have to standardize all the file paths and update imports. As far as I can tell this PR should be ok though

Reply:
Make sure to add a semicolon to import statement on line 5 as well